catroid/src/main/java/org/catrobat/catroid/utils/CrashReporter.java

```diff
@@ -29,61 +29,56 @@
 import android.support.annotation.VisibleForTesting;
 import android.util.Log;
 
-import com.crashlytics.android.Crashlytics;
-import com.crashlytics.android.core.CrashlyticsCore;
 import com.google.gson.Gson;
 
 import org.catrobat.catroid.BuildConfig;
 import org.catrobat.catroid.ui.SettingsActivity;
 
-import io.fabric.sdk.android.Fabric;
-
 public final class CrashReporter {
 
 	private static final String TAG = CrashReporter.class.getSimpleName();
 
-	private static SharedPreferences preferences;
 	public static final String EXCEPTION_FOR_REPORT = "EXCEPTION_FOR_REPORT";
+
+	public static CrashReporterInterface reporter = new CrashlyticsCrashReporter();
```

Comment:
Please make private and add a `@VisibleForTesting` setter.
I still don't know why `@VisibleForTesting` only works for methods and classes :worried: